src/Constants.ts

```diff
@@ -17,7 +17,7 @@ export class Constants {
     public static readonly BridgeDownloadDirectoryName = `bridge`;
     public static readonly KubectlDownloadDirectoryName = `kubectl`;
     public static readonly DotNetDownloadDirectoryName = `dotnet`;
-    public static readonly KubectlMinVersion = `1.21.2`;
+    public static readonly KubectlMinVersion = `1.27.3`;
```

Comment:
What's the effect of changing the version here? If I'm using an older version (like 1.26), it looks like it'll download and use a newer version (higher than min). Is that correct? And it'll just download somewhere for this extension, not overwriting anything in my PATH?

Reply:
Little correction - it would download the version of kubectl available from storage account and check if it's 1.27.3 if not it would error. Yes it would download to different path and wouldn't mess $PATH.